ansible/roles/osx-kvm/files/patch.git-diff

```diff
@@ -0,0 +1,110 @@
+diff --git a/OpenCore-Catalina/OpenCore.qcow2 b/OpenCore-Catalina/OpenCore.qcow2
```

Comment:
Umm, can we reconsider this? This is bad.

Reply:
You can always commit the resulting file instead.

Reply:
The "resulting file" is updated/generated from scratch frequently.

Is the functionality you are trying to provide provided by the following file?

https://github.com/kholia/OSX-KVM/blob/master/OpenCore-Catalina/OpenCore-nopicker.qcow2